### DISCUSSION TITLE
Shelly Wave 2PM dying after inclusion

### DISCUSSION THREAD

**Discussion by fcaputo** [zwave-js/zwave-js#6592] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
When I add a Shelly Wave 2PM it gets recognized properly, but dies after a couple of seconds and seems to reboot.

What did you expect to happen instead?
The device must not die.

### Software versions

Running Z-Wave JS UI as docker container on my Rapberry PI 5.

Driver (node-zwave-js): 12.4.1

Z-Wave JS UI: 9.6.2

### Device information

Manufacturer: Shelly 
Model name: Wave 2 PM
Node ID: 024

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[node024.log](https://github.com/zwave-js/node-zwave-js/files/13854733/node024.log)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
@fcaputo do you have all security keys configured? It looks like the S2 bootstrapping immediately fails, which should only happen when there are no keys to use.
